### PR TITLE
Refactor: Description messages on Admin and User resources

### LINKED ISSUE #577: Fix description messages on Backend Server
## Description
As a developer, i need to see the same way for displaying messages on the backend server. so that i can follow the same pattern on the messages im writing.

### Update 
The following methods should include:  
image before the message itself. 

Users
- [ ] GET / dashboard
- [ ] PUT /user
- [ ] POST user/send email
- [ ] GET /users
- [ ] GET /users/verified

Admins
- [ ]  GET /admins

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
2 hours

### PR DESCRIPTION
**Pull request by EfthymiaKostaki**
# Description
This pull request changes the description messages as stated in Admin and Users' resources to follow the general structure: {"message": "some message"}.

### The changes are these:
- Add specific messages on messages.py
- Refactor description messages on Admin resource
- Refactor description messages on User resource
- Remove related hardcoded messages. Add these messages to messages.py
so as to enable reuse of the same messages.

### The description APIs which have been changed are these:
- Admins: GET /admins
image
- Users: GET /users
image
- Users: GET /users/{user_id}
image

- Users: GET/users/verified
image

- Users: PUT /user
image

- Users: POST /user/resend_email
image

- Users: POST /refresh
image

- Users: POST/login
image

- Users: GET /home
image

- Users: GET /dashboard
image

### The description APIs which could not be changed are these:
- Users: GET user/
image

- Users: GET /users
image

- Users: GET /users/verified
image

Fixes #577

### Type of Change:
- Code
- User Interface

### How Has This Been Tested?

All tests on Travis have passed.

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**

- [x] My changes generate no new warnings

### CODE REVIEW

File: app/api/resources/user.py [hunk 1]
```diff
@@ -40,9 +40,9 @@ class UserList(Resource):
     @users_ns.doc("list_users", params={"search": "Search query"})
     @users_ns.doc(
         responses={
-            HTTPStatus.UNAUTHORIZED: f"{messages.TOKEN_HAS_EXPIRED['message']}<br>"
-            f"{messages.TOKEN_IS_INVALID['message']}<br>"
-            f"{messages.AUTHORISATION_TOKEN_IS_MISSING['message']}"
+            HTTPStatus.UNAUTHORIZED: f"{messages.TOKEN_HAS_EXPIRED}<br>"
```
Comment (satya7289) [inline app/api/resources/user.py:43]: Also add this for OK status `@users_ns.response(HTTPStatus.OK, f"{messages.GENERAL_SUCCESS_MESSAGE}", public_user_api_model)`.
And similary do for others like Users: GET user/ , Users: GET /users/verified by adding appropriate user api model.

**Comment by EfthymiaKostaki** [comment]
@satya7289 Hello, thanks for your comments. I am currently in exams period and I will try to implement the changes ASAP.

**Comment by EfthymiaKostaki** [comment]
@satya7289 Hey! Can you explain more specifically where I should implement the same OK status with the example you mentioned? Because in for example GET /users there is the HTTP status CREATED which is similar to HTTP  status OK and I am confused about what I should change since it is a change in the original code and not only in the message being shown.

**Comment by satya7289** [comment]
> @satya7289 Hey! Can you explain more specifically where I should implement the same OK status with the example you mentioned? Because in for example GET /users there is the HTTP status CREATED which is similar to HTTP status OK and I am confused about what I should change since it is a change in the original code and not only in the message being shown.

In the description of the PR, you have mentioned that `The description APIs which could not be changed are these:`. I am talking about these can be changed using `@users_ns.response(HTTPStatus.OK, f"{messages.GENERAL_SUCCESS_MESSAGE}", public_user_api_model)` similar type of coding(use an appropriate model as the 3rd argument of response).

**Comment by isabelcosta** [comment]
Thank you for contributing! I will close due to inactivity. You're always welcome to contribute again 🙌🏾 @EfthymiaKostaki

**Pull request closed without merging**